packages/localizations/src/chunks/other.en.ts

```diff
@@ -234,7 +234,7 @@ const otherEn =
       "numberOfCommon": "{{number}} common",
       "offerNotFound":
         "Offer not found. It might have been deleted by the author",
-      "inputPlaceholder": "Let’s write a message here...",
```

Comment:
And the same here, we need: `"inputPlaceholder": "Let’s write a message here..."`